ballerina/Ballerina.toml

```diff
@@ -0,0 +1,67 @@
+[package]
+org = "ballerina"
+name = "schema_registry"
+version = "0.1.0"
+authors = ["Ballerina"]
+export=["schema_registry"]
+keywords = ["schema_registry"]
```

Comment:
Keywords are not updated?

Reply:
Fixed in [e3a1f16](https://github.com/ballerina-platform/module-ballerinax-confluent.cregistry/pull/1/commits/e3a1f160756203db4476765f8c49d148e86e675a)